public/js/showoff.js

```diff
@@ -1689,7 +1689,7 @@ function setupStats(data)
 
 /* Is this a mobile device? */
 function mobile() {
-  return ( $(window).width() <= 480 )
+  return ( $(window).width() <= 640 )
```

Comment:
This is where the 480px came from originally, btw. https://responsivedesign.is/develop/browser-feature-support/media-queries-for-common-device-breakpoints/